package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@lambdatest/node-fetch-sessions",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/LambdaTest/node-fetch-session.git"
+  },
+  "author": "LambdaTest <keys@lambdatest.com>",
+  "license": "ISC",
```

Comment:
We have added Apache 2.0 license, so lets update it

Reply:
Done